OctoAwesome/OctoAwesome.Basics/EntityComponents/PositionComponent.cs

```diff
@@ -43,7 +39,9 @@ public override void Deserialize(BinaryReader reader, IDefinitionManager definit
             float posY = reader.ReadSingle();
             float posZ = reader.ReadSingle();
 
-            Position = new Coordinate(planet, new Index3(blockX, blockY, blockZ), new Vector3(posX, posY, posZ));
+            Position = new Coordinate(reader.ReadInt32(), 
+                new Index3(reader.ReadInt32(), reader.ReadInt32(), reader.ReadInt32()), 
+                new Vector3(reader.ReadSingle(), reader.ReadSingle(), reader.ReadSingle()));
         }
```

Comment:
Funktionier das, weil es wwurde ja schon gelesen

Reply:
Zum oberen ja es ist von dir, das wurde als neu registriert weil ich es von Basiscs in OctoAwesome verschoben habe.

Das zweite, warum sollte das nicht gehen. Beim Aufruf der Methode werden alle Parameter zusammen gesucht (sagen ich mal) und dafür muss halt ReadInt32() aufgerufen werden.